scripts/scald.rb

```diff
@@ -534,7 +534,9 @@ def hadoop_command
 end
 
 def jar_mode_command
-  "HADOOP_CLASSPATH=#{JARBASE} hadoop jar #{JARBASE} #{hadoop_opts} #{JOB} --hdfs " + JOB_ARGS
+  hadoop_libjars = ([MODULEJARPATHS.map{|n| File.basename(n)}].select { |s| s != "" }).flatten.join(",")
```

Comment:
jar mode i think expects things to be a fatjar. Adding modules just to the hadoop classpath, but not also on libjars is likely to cause it to fail on the mappers/reducers?